docs/source/modin/supported/index_supported.rst

```diff
@@ -126,7 +126,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``value_counts``            | P                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``set_names``               | P                               |                                  |                                                    |
```

Comment:
@sfc-gh-samdesai1 
 do you support either case of df.index and index's names changing when the other changes? If you've not done either or only done one, I suggest marking the documentation as P  and not Y  because the functionality is incomplete. Make a note of this in the documentation too.